tests/integ/test_tf_script_mode.py

```diff
@@ -105,7 +110,7 @@ def test_server_side_encryption(sagemaker_session, tf_full_version, py_version):
             train_instance_type="ml.c5.xlarge",
             sagemaker_session=sagemaker_session,
             script_mode=True,
-            framework_version=tf_full_version,
+            framework_version=tf_serving_version,
```

Comment:
why switch to using a serving version for training?

Reply:
because this test is doing a deploy later and TFS does not support 2.2 yet.